runtime/polkadot/src/lib.rs

```diff
@@ -766,6 +766,7 @@ impl pallet_membership::Config<pallet_membership::Instance1> for Runtime {
 parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = 100 * DOLLARS;
+	pub const ProposalBondMaximum: Balance = 1000 * DOLLARS;
```

Comment:
@rrtti let me know via Element that he wants to change the upper limit to the following:

```suggestion
	pub const ProposalBondMaximum: Balance = 500 * DOLLARS;
```